docs/tutorials/services/try-viam-color-detection.md

```diff
@@ -33,9 +33,23 @@ If you are [renting your rover](https://app.viam.com/try), we recommend that you
 Be aware that if you are running out of time during your rental, you can [extend your rover rental](/try-viam/reserve-a-rover/#extend-your-reservation) as long as there are no other reservations.
 {{< /alert >}}
 
+## Enable the cameras
+
+Before configuring color detection, enable the rover's camera to get a better sense of what it perceives as it performs color classification.
+
+* If you are running this tutorial with a [rented Viam Rover](https://app.viam.com/try), enable both provided cameras: the front-facing camera and the overhead cam.
+  In the `viam_base` component panel under the **Control** tab, enable both the `cam` for the front-facing camera and the `overhead-cam:overheadcam` for an overhead view of your rover.
+
+  ![Screenshot of the viam_base component panel showing both the 'cam' and 'overheadcam' camera feeds enabled.](try-viam/try-viam/enable-both-cameras.png)
```

Comment:
Alt texts should not beging with photo of or screenshot of, because then screenreaders will say photo of screenshot of
```suggestion
  ![The viam_base component panel showing both the 'cam' and 'overheadcam' camera feeds enabled.](try-viam/try-viam/enable-both-cameras.png)
```

Reply:
Also expanded this to 12 other instances across these two pages.